docs/software-installation-guide.md

```diff
@@ -3,45 +3,89 @@
 ## User installation
 
 (Optional but recommended) install [anaconda](https://www.anaconda.com/products/distribution) and create a virtual environment  
-```
+
+```sh
 conda create -y -n recOrder python=3.9
 conda activate recOrder
 ```
+
 Install `napari` and `recOrder-napari`:
-```
+
+```sh
 pip install "napari[all]" recOrder-napari
 ```
+
 Open `napari` with `recOrder-napari`:
-```
+
+```sh
 napari -w recOrder-napari
 ```
+
 View command-line help by running
-```
+
+```sh
 recOrder.help
 ```
 
-## Developer installation:
+## Developer installation
 
-Install [`git`](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git) and `conda` (either [anaconda](https://www.anaconda.com/products/distribution) or [miniconda](https://docs.conda.io/en/latest/miniconda.html)).
+### Windows (x86_64)
```

Comment:
and M1 `arm64` since these instructions have been working for me?

Reply:
Not exactly the behavior in my tests. I think we need more testing here to see what is the 'default' workflow (which may have changed due to variations in shipped OS setting, etc.)